static/js/apps/nl_eval/one_query_section.tsx

```diff
@@ -16,29 +16,29 @@
 
 import React, { useState } from "react";
 
-import { SearchResult } from "./search_result";
+import { ModelMatchBox } from "./model_match_box";
 import { BASE_URL, EmbeddingObject } from "./util";
 
-export interface SentenceSectionProps {
+export interface OneQuerySectionProps {
   sentence: string;
   modelNames: string[];
   goldenStatVars: string[];
   customDescription: Record<string, EmbeddingObject[]>;
 }
 
-export function SentenceSection(props: SentenceSectionProps): JSX.Element {
+export function OneQuerySection(props: OneQuerySectionProps): JSX.Element {
```

Comment:
nit: curious why OneQuerySection, will there be another MultiQuerySection? If not, could this just be QuerySection?

Reply:
Updated